html/browsers/sandboxing/resources/broadcast-done.sub.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<meta charset=utf-8>
+<script>
+  const bc = new BroadcastChannel({{GET[uuid]}});
```

Comment:
Safari fails because of the use of BroadcastChannel here. Is there an alternative approach that might work for this test?

Reply:
The opener and the openee are from the same browsing context group. If the popup was loaded, you could have used opener.top.postMessage().
Moreover, they are even same-origin, so, you also would have direct access to it.

Reply:
I could use wptserve stash, but it gets complicated for something that nobody fails. We could remove this check altogether, which also avoids the timeout. WDYT?

Reply:
@ArthurSonzogni The test already checks that `window.open` returns null (because of `sandbox`). If an incorrect implementation still opens a popup, but with `opener` being null, I wanted to fail the test. But maybe not worth it

Reply:
Actually using stash wasn't so complicated here. Done in 7a3388bd52f2ba08d585301721752266d2737cd9

Reply:
Okay, so you wanted to ensure that if the popup loaded AND had a null opener, then test still fails properly.
Indeed using stash seems to be the preferable option.

Reply:
Yes, exactly. Can either of you approve this PR, please?